recipes/cppfront/all/conanfile.py

```diff
@@ -0,0 +1,93 @@
+from conan import ConanFile
+
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import copy, get, rmdir
+
+import os
+
+required_conan_version = ">=1.52.0"
+
+class CppfrontConan(ConanFile):
+    name = "cppfront"
+    description = "Cppfront is a experimental compiler from a potential C++ 'syntax 2' (Cpp2) to today's 'syntax 1' (Cpp1)"
+    topics = ("cppfront", "cpp2")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/hsutter/cppfront"
+    license = "CC-BY-NC-ND-4.0"
+    settings = "os", "arch", "compiler", "build_type"
+
+    @property
+    def _compilers_minimum_version(self):
+        return {"gcc": "11",
+                "Visual Studio": "16.9",
```

Comment:
This is missing support for the new compiler `msvc` you can check the template https://github.com/conan-io/conan-center-index/blob/010c142bd05ae6a29667797010ecf70c5040dc85/docs/package_templates/cmake_package/all/conanfile.py#L90

for the details on how to sneak this is

Reply:
```suggestion
                "Visual Studio": "16.9", # required for finer precision 
```